src/library_nodepath.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 The Emscripten Authors
+ * SPDX-License-Identifier: MIT
+ */
+
```

Comment:
Maybe add a comment in this file explaining the motivation for it? (which IIUC from the discussion is windows?) lgtm with that.

Reply:
Added a comment with 37b6dd0c4e7d66b0daf79b6d2fcbbb041a01dd98. I also removed the "This has mostly been tested on Linux so far"-phrase from `NODERAWFS` settings description with 911505295b60ff3e45c4203b3aed262b09eff0c2, since using Node.js operations should be cross-platform (+ AFAIK,  CI also test the `NODERAWFS` setting on OSes other than Linux).

Reply:
Thanks!